### DISCUSSION TITLE
onEvent EventRefs - can the event ref be repeated?

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#519] [Q&A] [answered]
in the OnEvent definiition, eventRefs is defined to be an array.  can the same eventRef occur more than once in the array?  

Eg. "eventRefs": ["_HighBodyTemperature_", "HighBloodPressure", "_HighBodyTemperature_", "_HighBodyTemperature_"],

**Comment by tsurdilo** [comment, marked as answer]
No the idea is that this array should only include unique items. The schema for eventRefs is missing
"uniqueItems": true

will add.
Thank you for bringing this up!

**Reply by tsurdilo** [reply]
https://github.com/serverlessworkflow/specification/issues/520

**Reply by tsurdilo** [reply]
PR: https://github.com/serverlessworkflow/specification/pull/521
